content/blog/2018-05-02-onboarding-is-work.md

```diff
@@ -0,0 +1,325 @@
+could be due to different age at submission. How old are packages at
+submission?
+
+``` r
+age <- commits %>%
+  dplyr::group_by(package) %>%
+  dplyr::summarise(age = difftime(min(opened), min(datetime), units = "weeks")) %>%
+  dplyr::filter(age > 0)
+
+library(hrbrthemes)
+ggplot(age) +
+  geom_dotplot(aes(age), binwidth = 5) +
+  xlab("Age (weeks)") +
+  scale_y_continuous(NULL, breaks = NULL) +
+  hrbrthemes::theme_ipsum(base_size = 16,
+                          axis_title_size = 16)
+```
+
+![age of packages at
+submission](/img/blog-images/2018-05-02-onboarding-is-work/unnamed-chunk-4-1.png)
+
+This shows *apparent* age because the first commit might happen a long
+time after the package was created, although often one makes an initial
+commit not long after having started to work. I needed to filter
+positive age because in one case the GitHub repo of the onboarded
+package was apparently deleted and re-created without history after
+approval. Such things happen, thankfully the package wasn’t lost, just
+its history!
+
+Many packages are submitted quite rapidly which might indicate rapid
+development, but also maybe onboarding as a part of development,
+i.e. authors knew they intended to submit. For instance, Maëlle’s
+developed `opencage` in just a few days, then submitted it to get it
+ready for wider use by the community.
+
+### Work done by reviewers
+
+How much effort is put in by reviewers in this process? Reviewer time
+and effort is one of our most previous resources (you can read more
+about reviewer motivations and perspectives in other blog posts by [Mara
+Averick](https://ropensci.org/blog/2017/08/22/first-package-review/),
+[Verena
+Haunschmid](https://ropensci.org/blog/2017/09/08/first-review-experiences/),
+[Charles T. Gray](https://ropensci.org/blog/2018/03/13/ode-to-testing/)
+and [Miles
+McBain](https://ropensci.org/blog/2018/04/06/peer-review-value/). We
+this effort by asking reviewers to self-report the hours spent
+reviewing. We do so to understand the amount of effort we are asking
+reviewers for, so we can let new reviewers know what to expect, and so
+hopefully in the future we can measure the success of efforts to
+automate some reviewer tasks.
+
+``` r
+# airtable data
+# this is our private database of who's reviewed what
+# and of reviewers' areas of expertise
+airtable <- airtabler::airtable("appZIB8hgtvjoV99D", "Reviews")
+airtable <- airtable$Reviews$select_all()
+```
+
+We have self-reported reviewing times for 136 of 184 reviews.
+
+``` r
+ggplot(airtable) +
+  geom_boxplot(aes(y = review_hours, x = "")) +
+  hrbrthemes::theme_ipsum(base_size = 16,
+                          axis_title_size = 16) +
+  theme(axis.title.x=element_blank()) +
+  ylab("Reviewing time (hours)")
+```
+
+![hours spent
+reviewing](/img/blog-images/2018-05-02-onboarding-is-work/unnamed-chunk-6-1.png)
+
+As we found before, it appears to take our reviewers a similar amount of
```

Comment:
"As we found before"

where? when? reference?

Reply:
I think there's no ref besides @noamross' unpublished post 🤔

Reply:
will remove that part of the sentence